base/doc/ltnews41.tex

```diff
@@ -1078,30 +1282,45 @@ \subsection{\pkg{array}:\ Fix handling of empty p-cells}
 %
 \githubissue{1622}
 
+
 \section{Changes to files in the L3 programming layer}
 
-Work on the L3 programming layer continues in parallel with development of the
-\LaTeX{} kernel. Of note for developers is that we have integrated more code
+%% Should this use subsections? 
```

Comment:
Maybe: I originally just wrote one paragraph, then got asked to add other things :)